src/test/resources/pages/godmode.html

```diff
@@ -56,4 +56,4 @@ <h2>Unpredictable values</h2>
             </ul>
         </div>
     </body>
-</html>
+</html>\n
```

Comment:
@wkurniawan07 

It seems that "\n" is not working. I am trying it with "<br>" instead